diagnostic.js

```diff
@@ -30,25 +30,53 @@ mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
 
-const create = (name, description, startYear, endYear) => {};
+const create = (name, description, startYear, endYear) => {
+  Movement.create({
+    name: name,
+    description: description,
+    startYear: startYear,
+    endYear: endYear
+  }).then(movement => {
+    console.log(movement.toJSON())
+  }).catch(console.error);
+};
+
+
 // Success -> Print new Movement as JSON
 // Failure -> Console.error
 
-const index = () => {};
+const index = () => {
+  Movement.find()
+  .then(movements => {
+    console.log(movements.toJSON())
```

Comment:
Almost. `movements` is an array, but the `.toJSON` method there is only available to Mongoose documents.